sql/expression/function/date_format.go

```diff
@@ -0,0 +1,316 @@
+package function
+
+import (
+	"fmt"
+	"github.com/lestrrat-go/strftime"
+	"github.com/liquidata-inc/go-mysql-server/sql"
+	"github.com/liquidata-inc/go-mysql-server/sql/expression"
+	"strconv"
+	"time"
+)
+
+func panicIfErr(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func monthNum(t time.Time) string {
+	return strconv.FormatInt(int64(t.Month()), 10)
+}
+
+func dayWithSuffix(t time.Time) string {
+	suffix := "th"
+	day := int64(t.Day())
```

Comment:
uint8?

Reply:
Why this cast at all, really? Why not just leave it int?

Reply:
Has to be cast to pass to FormatInt.